### ISSUE TITLE
Identity first-time contributors and external contributors

### ISSUE THREAD

**Issue by pjcozzi** [AnalyticalGraphicsInc/cesium-concierge#85]
We like to tweet to congratulate and thank first-time contributors and external contributors.  It would be useful for a comment to explicitly `@mention` the outreach team so they know to tweet.

Currently, it is possible to identify first-time contributors by filtering emails for "Welcome to the Cesium community", but I don't know if there is a unique way to filter for external contributors, but this would be easy for the concierge to check.

CC @slchow @vanessamoore

**Comment by pjcozzi** [comment]
Bumping this since we like to tweet about all community contributions.

When a pull request from a community contributor is merged, the concierge could just @ mention someone(s) (configurable) and also say if it is a first time contributor.

Longer-term, if/when the concierge can label issues, it could label issues as "community contribution", "first-time contributor", etc. - which would be useful for stats as well.

**Comment by pjcozzi** [comment]
@OmarShehata is this one on the short-term radar?

**Comment by OmarShehata** [comment]
Yes, I ran into the same problem of checking the correct branch/fork as in this PR https://github.com/AnalyticalGraphicsInc/cesium-concierge/pull/118 so it'll be easy to do right after that one.

**Comment by OmarShehata** [comment]
This is merged, and I have seen concierge reporting "Thanks for your first time contribution!" need to investigate why it didn't post for this one https://github.com/AnalyticalGraphicsInc/cesium/pull/7944.

**Comment by verybigzhouhai** [comment]
@OmarShehata Is it because the first PR I submitted before was shut down by myself, so it was not identified this time?  https://github.com/AnalyticalGraphicsInc/cesium/pull/7911，When I shut down the pr, I received a message from the concierge.

**Comment by OmarShehata** [comment]
I think it may be an error in my logic, that Concierge checks the branch you're merging into to see if your name is in the `contributors.md`, and decides based on that whether you're a new contributor. 

But I think what happened here, is that since Concierge checks _after_ the PR is merged, then your name is already in `contributors.md` in master, so it doesn't post. I think I need to check for the commit when the PR was opened, not when the PR is merged.
